### PR TITLE
Update Screen Captures in Kibana Logging on Astronomer Software

### LINKED ISSUE #700: kibana-logging.md out of date
**Affected Document:**
[kibana-logging.md](https://github.com/astronomer/docs/blob/main/software/kibana-logging.md)

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**
The Kibana dashboard to create `index pattern`s has changed

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #700

### CODE REVIEW

**Comment by RNHTTR** [comment]
I think it might be helpful to say something like...
> Open the hamburger menu and navigate to `Stack Management`. From there, click on `Index Patterns` --> `Create index pattern` ...

Also maybe placing that instruction above the screenshot?

**Review by RNHTTR — APPROVED** [review]
LGTM!

**Comment by kwcanuck** [comment]
@jwitz ready for your review.

**Review by jwitz — APPROVED** [review]
one minor comment, otherwise LGTM!

File: software/kibana-logging.md [hunk 1]
```diff
@@ -9,24 +9,29 @@ All Airflow logs from your Astronomer logs will flow to Elasticsearch and can be
...
 
-![Index Pattern](https://assets2.astronomer.io/main/docs/ee/index_pattern.png)
+2. Click **Menu** in the upper left, click **Stack Management**, and then click **Index Patterns**.
 
+3. In the **Name** field enter `fluentd.*` and then click **Create index pattern**.
 
-Navigate to `Management` and create an index pattern for `fluentd.*`
-Elasticsearch uses [index patterns](https://www.elastic.co/guide/en/kibana/current/index-patterns.html) to organize how you explore data. Setting `fluentd.*` as the index means that Kibana will display all logs from all deployments (Astronomer uses `fluentd` to ship logs from pods to ElasticSearch).
+    <div class="text--center">
+    <img src="/img/docs/index-pattern.png" alt="Elastic Create Index Pattern screen" />
```
[the remark this answers is not in the document]
Comment (kwcanuck) [reply software/kibana-logging.md:19]: @jwitz updated screenshot to just the Create index pattern window

**Pull request merged**